sabakan/repairer.go

```diff
@@ -2,18 +2,27 @@ package sabakan
 
 import (
 	"strings"
+	"time"
 
 	"github.com/cybozu-go/cke"
 	"github.com/cybozu-go/log"
 )
 
-func Repairer(machines []Machine, entries []*cke.RepairQueueEntry, constraints *cke.Constraints) []*cke.RepairQueueEntry {
+func Repairer(machines []Machine, repairEntries []*cke.RepairQueueEntry, rebootEntries []*cke.RebootQueueEntry, constraints *cke.Constraints, now time.Time) []*cke.RepairQueueEntry {
 	recent := make(map[string]bool)
-	for _, entry := range entries {
+	for _, entry := range repairEntries {
 		// entry.Operation is ignored when checking duplication
 		recent[entry.Address] = true
 	}
 
+	rebootLimit := now.Add(time.Duration(-constraints.RepairRebootingSeconds) * time.Second)
```

Comment:
I feel like I've asked this before 🙇 , but why are you subtracting repairRebootingSeconds from the current time here?

Reply:
Thank you for your detailed explanation.
It has helped me to understand.